tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/AmqpSendReceiveInterceptorTest.java

```diff
@@ -26,8 +26,12 @@
 import org.apache.activemq.transport.amqp.client.AmqpReceiver;
 import org.apache.activemq.transport.amqp.client.AmqpSender;
 import org.apache.activemq.transport.amqp.client.AmqpSession;
+import org.apache.felix.resolver.util.ArrayMap;
```

Comment:
31 commits ;) wowsies you work fast :P

I think you may have accidentally brought in a lot of other commits in from your local repo, when doing your update :)